ArduCopter/Copter.cpp

```diff
@@ -512,6 +512,10 @@ void Copter::one_hz_loop()
     adsb.set_is_flying(!ap.land_complete);
 #endif
 
+#if PARACHUTE == ENABLED
+    parachute.set_is_flying(!ap.land_complete);
```

Comment:
totally missed from https://github.com/ArduPilot/ardupilot/pull/9506/files so nice to have it !